src/app/issues-viewer/issues-viewer.component.ts

```diff
@@ -96,25 +104,27 @@ export class IssuesViewerComponent implements OnInit, AfterViewInit, OnDestroy {
   }
 
   /**
-   * Update the list of hidden user based on the new info.
-   * @param issueLength The number of issues assigned to this user.
-   * @param assignee The assignee.
+   * Update the list of hidden group based on the new info.
+   * @param issueLength The number of issues under this group.
+   * @param group The group.
    */
-  updateHiddenUsers(issueLength: number, assignee: GithubUser) {
-    if (issueLength === 0) {
-      this.updateHiddenUser(assignee);
+  updateHiddenGroups(issueLength: number, target: Group) {
+    if (issueLength === 0 && this.groupingContextService.isInHiddenList(target)) {
+      this.updateHiddenGroup(target);
     } else {
-      this.removeHiddenUser(assignee);
+      this.removeHiddenGroup(target);
     }
   }
 
-  private updateHiddenUser(assignee: GithubUser) {
-    if (!this.hiddenAssignees.includes(assignee)) {
-      this.hiddenAssignees.push(assignee);
+  private updateHiddenGroup(target: Group) {
+    const isGroupPresent = this.hiddenGroups.some((group) => group.equals(target));
+
+    if (!isGroupPresent) {
+      this.hiddenGroups.push(target);
     }
   }
 
-  private removeHiddenUser(assignee: GithubUser) {
-    this.hiddenAssignees = this.hiddenAssignees.filter((user) => user !== assignee);
+  private removeHiddenGroup(target: Group) {
+    this.hiddenGroups = this.hiddenGroups.filter((group) => !group.equals(target));
```

Comment:
The naming of these functions could be made clearer. As they are they imply updating the group and removing the group but is unclear what the update is or what the removal is from.
A clearer name would be something like ``addToHiddenGroups`` and ``removeFromHiddenGroups``